content/conditions/diarrhoea/main-content.md

```diff
@@ -0,0 +1,142 @@
+## How you can treat diarrhoea yourself 
+
+Diarrhoea should go away on its own within a few days. It’s normal to also have stomach cramps and headaches. 
+
+It’s important that you drink enough fluids to [avoid dehydration](http://www.nhs.uk/Conditions/dehydration/Pages/Introduction.aspx).
+Your pee should be light yellow or clear.
+
+Drink small sips of things like water mixed with juice and soup broth to make sure you get enough salt and sugar. 
+
+As soon as you feel able to eat, try things like potatoes, rice, bananas, soup, boiled vegetables or salty foods. 
+
+To help with headaches or fevers you can take painkillers like paracetamol or ibuprofen. 
+
+
+## Your pharmacist can help with diarrhoea  
+
+You can take medicines to help reduce the diarrhoea and shorten how long it lasts. However, you don’t have to
+take these and they won’t cure your diarrhoea. 
+
+Describe your symptoms to your pharmacist. They can recommend the best medicine for you.
+
+Frail or elderly people can dehydrate more easily. You can get sachets with salt, sugar and minerals at the pharmacy
+that will help you stay hydrated. They’re called oral rehydration solutions. 
+
+!!! info_compact
+  [Find your nearest pharmacy](https://beta.nhs.uk/finders/find-help) 
+!!!
+
+!!! attention
+  ## See a GP if:
```

Comment:
Also, we should be consistent with `your` and `a`. Will need to check previous conditions for this.